pkg/licensing/expression/category.go

```diff
@@ -358,4 +365,126 @@ var (
 		Unlicense,
 		ZeroBSD,
 	}
+
+	// SpdxLicenseExceptions contains all supported SPDX Exceptions
+	// cf. https://spdx.org/licenses/exceptions.json
+	// used `awk -F'"' '/"licenseExceptionId":/ {print toupper("\"" $4 "\"," )}' exceptions.json ` command
```

Comment:
Since we need to keep it up-to-date, it should be done by `mage spdx` or something like that. I think we should create a separate file for the list and add 
```
// Code generated by "mage spdx", DO NOT EDIT.
// source: https://spdx.org/licenses/exceptions.json
```

to the header.

Reply:
Do you want to check `exceptions.json` file in tests?
I mean the same as for [mage docs:generate](https://github.com/aquasecurity/trivy/blob/17827db6a963232f3d5c759065518de6fdb591c3/.github/workflows/test.yaml#L61C1-L68C55)


This will help keep the file up-to-date, but can be noisy for PRs when a new version of file is released.
On the other hand, we can add a separate action to check the file's relevance once a week.

Reply:
Using `curl` and `awk` through `go generate` is the easiest way, but some environments don't have `curl`, and CLI flags might be different. Ideally, we should do that in Go.

Reply:
>Do you want to check exceptions.json file in tests?

No, we don't need it. We can update the file when we notice that.

Reply:
yes, that's what i planned
this command (using `awk` and other commands) is just a quick way to get all exceptions